src/libraries/Microsoft.Extensions.DependencyInjection/tests/DI.Tests/ServiceProviderDefaultContainerTestsWithOptions.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.Extensions.DependencyInjection.Specification;
+
+namespace Microsoft.Extensions.DependencyInjection.Tests
+{
+    public class ServiceProviderDefaultContainerTestsWithOptions : DependencyInjectionSpecificationTests
+    {
+        protected override IServiceProvider CreateServiceProvider(IServiceCollection collection)
+        {
+            try
+            {
+                return collection.BuildServiceProvider(ServiceProviderMode.Default, new ServiceProviderOptions
+                {
+                    ValidateOnBuild = true,
+                    // Too many tests fail because they try to resolve scoped services from the root
+                    // provider
+                    // ValidateScopes = true
+                });
+            }
+            catch
```

Comment:
Couldn't find a cleaner way to do that.

Reply:
Catch a specific exception type? (So other errors fail the test?)

Reply:
AggregateException is what this typically throws. I'll do that